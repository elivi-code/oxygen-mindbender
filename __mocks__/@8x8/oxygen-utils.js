export * from '@8x8/oxygen-utils';

const useFontSize = jest.fn(() => 16);

export { useFontSize };
