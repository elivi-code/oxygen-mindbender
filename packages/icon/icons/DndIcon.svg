<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
<rect x="19.2" y="9.6001" width="4.8" height="14.4" rx="2" transform="rotate(90 19.2 9.6001)" fill="#5E6D7A"/>
</svg>
