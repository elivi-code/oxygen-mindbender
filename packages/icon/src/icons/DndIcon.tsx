import React from 'react';
import type { IconProps } from '../utils/createSvgIcon';
import createSvgIcon from '../utils/createSvgIcon';

export type { IconProps };
export default createSvgIcon(<g><rect width="4.8" height="14.4" x="19.2" y="9.6001" rx="2" transform="rotate(90 19.2 9.6001)"/></g>, 'DndIcon');
