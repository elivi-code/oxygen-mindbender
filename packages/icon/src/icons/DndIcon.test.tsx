/* eslint-disable */
import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import { icon as theme } from '@8x8/oxygen-constants';
import DndIcon from './DndIcon';

describe('<DndIcon>', () => {
  let wrapper;

  function setup(props = {}) {
    return renderer.create(<DndIcon theme={theme} {...props} />).toJSON();
  }

  it('should render correctly', () => {
    wrapper = setup();
    expect(wrapper).toMatchSnapshot();
  });
});
/* eslint-enable */
