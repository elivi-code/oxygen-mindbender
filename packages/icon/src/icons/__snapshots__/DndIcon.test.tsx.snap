// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DndIcon> should render correctly 1`] = `
.c0 {
  height: 24px;
  width: 24px;
  min-height: 24px;
  min-width: 24px;
  -webkit-user-select: auto;
  -moz-user-select: auto;
  -ms-user-select: auto;
  user-select: auto;
  pointer-events: auto;
  fill: #1C2025;
}

<svg
  className="c0"
  color={null}
  disabled={false}
  role="presentation"
  selected={false}
  size={null}
  viewBox="0 0 24 24"
>
  <g>
    <rect
      height="14.4"
      rx="2"
      transform="rotate(90 19.2 9.6001)"
      width="4.8"
      x="19.2"
      y="9.6001"
    />
  </g>
</svg>
`;
